developers/weaviate/client-libraries/typescript/typescript-v3.md

```diff
@@ -399,87 +348,34 @@ Only Read operations powered by GraphQL.
-
-### Run a TypeScript file
-
-To run a TypeScript file, first convert it to JavaScript. The `typescript` package from `npm` includes the `tsc` utility. Use `tsc` to convert (transpile) the TypeScript file. 
-
-Install the `typescript` package. Add the `-g` flag if you want the package to be available globally.
-
-```bash
-npm install typescript
-```
-You can then use this command to transpile the TypeScript file. 
-
-```bash
-tsc
-```
-
-`node` only allows the `import` statement in modules. To allow the `import` statement, add the following to your `package.json` file.
-
-```json
-{ 
-   "type": "module"
-}
-```
-
-### Example
-
-To run this example, complete these steps. 
+## What's Next
 
-- Install the `typescript` package. 
-- Update the `tsconfig.json` and `package.json` files as described above.
-- Copy the sample code.
-- Save the code as `sample.ts` in the same directory as `tsconfig.json` and `package.json`.
-- Convert and run the code.
+We have curated some resources to help you get started using the client. 
 
-Use this code to create `sample.ts`.
 
-<details>
-  <summary>Sample TypeScript code</summary>
+### Recipes
 
-```ts
-import weaviate, { WeaviateClient } from 'weaviate-client'
+We have a repository of code recipes that follow popular Weaviate use cases. 
+You can run these on the [recipes-ts](https://github.com/weaviate/recipes-ts) Github repository.
 
-const client: WeaviateClient = await weaviate.connectToWCS(
-  'some-endpoint.weaviate.network', {
-    authCredentials: new weaviate.ApiKey('api-key'),
-  } 
-)
- 
-console.log(await client.isReady())
-```
-</details>
+### Demo Applications
 
-Convert `sample.ts`.
+We have a few demo applications writtten in Typescript and Javascript 
 
-```bash
-tsc 
-```
+- [QuoteFinder](https://github.com/weaviate/quote-finder/tree/main)
+- [NuxtVectorSearch](https://github.com/malgamves/nuxt-typescript-vector-search)
 
-Run the converted file.
+:::note v2 → v3 client documentation migration
+We are in the process of migrating our v2 client code examples to v3 as necessary. Please be patient as we work on documenting the rest.
+:::
 
-```bash
-node sample.js  
-```
+At the moment, we've covered the following pages. 
 
-The output looks like this.
+- [Introduction](https://weaviate.io/developers/weaviate/introduction)
+- [Quickstart](https://weaviate.io/developers/weaviate/quickstart)
+- [Starter Guides](https://weaviate.io/developers/weaviate/starter-guides)
+- [How-to: Search](https://weaviate.io/developers/weaviate/search)
+- [How-to: Manage Data](https://weaviate.io/developers/weaviate/manage-data)
 
-<details>
-  <summary>Sample output</summary>
 
-```json
-{
-  "clientId": "wcs",
-  "href": "https://auth.wcs.api.weaviate.io/auth/realms/SeMI/.well-known/openid-configuration"
-}
-```
-</details>
+Our [RESTful endpoints](../../api/rest/index.md) and [GraphQL functions](../../api/graphql/index.md) covered by the TypeScript client currently have JavaScript examples in the code blocks.
```

Comment:
The Questions and feedback include is missing

The Client releases include is also missing, but that might be ok for now since the v3client isn't actually released yet.